File: web/testit.py

import subprocess 



for x in range(10):
    subprocess = subprocess.Popen("curl -s -w '%{time_total}' -o /dev/null http://localhost:3000", shell=True, stdout=subprocess.PIPE)
    subprocess_return = subprocess.stdout.read()
    print(subprocess_return)

File: web/testit.go
package main

import (
	"log"
	"os/exec"
)

func main() {
	cmd := exec.Command("curl -s -w '%{time_total}' -o /dev/null https://localhost:3000")
	log.Printf("Running command and waiting for it to finish...")
	err := cmd.Run()
	if err != nil {
		log.Printf("Command finished with error: %v", err)
	}
}
